packages/rich-text/src/plugins/EmbeddedResourceInline/FetchingWrappedResourceInlineCard.tsx

```diff
@@ -54,7 +54,10 @@ export function FetchingWrappedResourceInlineCard(props: FetchingWrappedResource
     defaultTitle: 'Untitled',
   });
   const truncatedTitle = truncateTitle(title, 40);
-  const status = getEntryStatus(entry.sys, props.sdk.field.locale);
+  const status = getEntityStatus(
+    entry.sys,
+    props.sdk.parameters.instance.useLocalizedEntityStatus ? props.sdk.field.locale : undefined
```

Comment:
Are we still planning to later introduce something in the spirit of `sdk.entity.getStatus(sys: EntrySys)` that houses the conditional logic?

Reply:
yes, still waiting for an answer and will then create a follow up ticket for this 👍